subgraph/flake.nix

```diff
@@ -0,0 +1,47 @@
+{
+  description = "Flake for development orderbook subgraph workflows.";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
```

Comment:
let's pick a specific commit

Reply:
Ok, it's using commit: `ec750fd01963ab6b20ee1f0cb488754e8036d89d`